### PR TITLE
Handle network issues in the remote-data-provided example

### PR DESCRIPTION
**Pull request by limonte**
Fixes https://github.com/vaadin/px-data-grid/issues/112

### CODE REVIEW

File: demo/px-data-grid-remote-data-provider-demo.html [hunk 1]
```diff
@@ -157,6 +157,10 @@
             callback(JSON.parse(xhr.responseText), size);
           };
 
+          // In case of network issues, pass the empty array
```
Comment (toshegg) [inline demo/px-data-grid-remote-data-provider-demo.html]: `the empty array` => `an empty array`

File: demo/px-data-grid-remote-data-provider-demo.html [hunk 2]
```diff
@@ -157,6 +157,10 @@
...
+          // When network will be good, px-data-grid will try to get the same chunk of data again
```
Comment (toshegg) [inline demo/px-data-grid-remote-data-provider-demo.html]: `When network will be good` => `When network is good`

File: demo/px-data-grid-remote-data-provider-demo.html (hunk 1 above)
Comment (limonte) [reply demo/px-data-grid-remote-data-provider-demo.html]: done.

File: demo/px-data-grid-remote-data-provider-demo.html (hunk 2 above)
Comment (limonte) [reply demo/px-data-grid-remote-data-provider-demo.html]: done.

**Pull request merged**